internal/internal_workers_test.go

```diff
@@ -899,3 +877,17 @@ func (s *WorkersTestSuite) TestMultipleLocallyDispatchedActivity() {
 	s.True(activityResponseCompletedCount > 0)
 	s.True(activityCalledCount > 0)
 }
+
+// wait for test to complete - timeout and fail after 10 seconds to not block execution of other tests
+func startWorkerWaitForChannelAndTimeoutStopWorker(s *WorkersTestSuite, worker *aggregatedWorker, doneCh *chan struct{}) {
```

Comment:
nit: just `startWorkerAndWait` would be OK. Currently the name captures **everything** that function does, which makes it too long.